ts/TreeUtil.ts

```diff
@@ -70,7 +70,11 @@ export class TreeUtil {
             });
         }
     }
-
+    async cutFix() {
+        this.railroad.removedVegetationAssets = [];
+        this.railroad.vegetation = [];
+        this.setMapModified();
+    }
```

Comment:
The TreeUtil is intended for use with the Pine Valley trees. It is named in a generic way because it predates Lake Valley, but I don't think this file should modify `vegetation`. Can you elaborate on what you're trying to solve here - do you just want a way to replant all trees in Lake Valley?

Reply:
My save file i used for testing is pine valley and atm if i cut all trees with current live version of studio and load it to the game it will not me replant all trees after i save the game so thats why the cut fix is there

Reply:
Also after cutting all trees save file goes to 5mb what is expected but after the 5mb save is loaded in game and saved it goes up to 10mb and corrupts the save for use in studio